cli/medperf/comms/rest.py

```diff
@@ -31,6 +31,18 @@ def login(self, ui: UI):
         else:
             self.token = res.json()["token"]
 
+    def change_password(self, pwd: str):
+        """Sets a new password for the current user.
+
+        Args:
+            pwd (str): New password to be set
+            ui (UI): Instance of an implementation
+        """
+        body = {"password": pwd}
+        res = self.__auth_post(f"{self.server_url}/me/password/", json=body)
+        if res.status_code != 200:
+            pretty_error("Unable to change the current password", self.ui)
+
```

Comment:
Right now, we're relying on a specific detail of this implementation to prevent deleting the credential in the case of this API failing. I think it would be best if this interface explicitly returned a boolean for us to check, even if in our current implementation it will never matter (because we error out here).

Reply:
Thanks Micah, this makes sense. Checking both sides ensures that future comms implementations that might not explicitly error out will be handled correctly either way